src/main/java/com/smartbear/readyapi/client/execution/Execution.java

```diff
@@ -180,8 +194,13 @@ public TestStepResultReport getLastTestStepResult(String testStepName) {
         }
 
         @Override
-        public TestStepResultReport getTestStepResult(int index) {
+        public TestStepResult getTestStepResult(int index) {
             return results.get(index);
         }
+
+        @Override
+        public Execution getExecution() {
```

Comment:
I don't see this being used anywhere. I think we are complecting it a bit. ProjectRecipeExecutionResult  is accessed through an Execution object, so client already has execution reference, why do we need this method?